code/modules/spells/spell_types/touch/scream_for_me.dm

```diff
@@ -16,20 +16,49 @@
 	hand_path = /obj/item/melee/touch_attack/scream_for_me
 
 /datum/action/cooldown/spell/touch/scream_for_me/on_antimagic_triggered(obj/item/melee/touch_attack/hand, mob/living/victim, mob/living/carbon/caster)
-	caster.visible_message(
-		span_warning("The feedback mutilates [caster]'s arm!"),
-		span_userdanger("The spell bounces from [victim]'s skin back into your arm!"),
+
+
+	// Informs the wizard of the backfire
+	to_chat(victim, span_warning("The feedback mutilates [caster]'s arm!"))
+	to_chat(caster, span_userdanger("The spell bounces from [victim]'s skin back into your arm!"))
```

Comment:
``visible_message`` is supposed to do what you changed it into, there's no reason for this change